feature_engine/transformation/log.py

```diff
@@ -13,7 +13,7 @@
 
 class LogTransformer(BaseNumericalTransformer):
     """
-    The LogTransformer() applies the natural logarithm or the base 10 logarithm to
+    The LogTransformer() applies the natural logarithm, the base 10 logarithm or exponetial function to
```

Comment:
Could we revert to the original line in the docstring? Normally we describe the transformers by their functionality in the transform method and not in the inverse_transform. It is a convention :)

Reply:
Would you please clarify on this request? New transform funcionality is explained same way it used to be: The Class used to described about the logarithim part and I just added the exponential. New method is explained on the new method, same way old one. I mean, if I revert still "natural logarithm and base 10" will be there, which as you mentioned dont follow the convention.